x.mjs

```diff
@@ -22,6 +22,37 @@ program
     await $`pnpm install`;
   });
 
+// x clean
+let cleanCommand = program
+  .command("clean")
+  .description("clean node_modules/ and target/ directory");
+
+// x clean all
+cleanCommand.command("all").action(async function () {
+  await $`./x clean rust`;
+  await $`./x clean node`;
+});
+
+// x clean node
+cleanCommand
+  .command("node")
+  .description("Clean node_modules")
+  .action(async function () {
+    await $`rm -rf node_modules`;
```

Comment:
No idea on how to do this in across platform, `fs.rm` seems really slow

Reply:
@hardfist Any ideas for clean up `node_modules`?

Reply:
https://github.com/microsoft/rushstack/blob/e9d701dd382d7139dc21651ace24a36589650557/libraries/rush-lib/src/utilities/AsyncRecycler.ts#L104 use this

Reply:
This is too complicated, let me delete this part of the script for now..